buildutils/src/ensure-repo.ts

```diff
@@ -37,7 +37,19 @@ const UNUSED: Dict<string[]> = {
   '@jupyterlab/buildutils': [
     '@babel/core',
     '@babel/preset-env',
-    'babel-loader'
+    'babel-loader',
+    'css-loader',
+    'file-loader',
+    'html-loader',
+    'raw-loader',
+    'source-map-loader',
+    'style-loader',
+    'svg-url-loader',
+    'terser-webpack-plugin',
+    'to-string-loader',
+    'url-loader',
+    'webpack-cli',
+    'worker-loader'
```

Comment:
You added a lot more packages than I did: https://github.com/jupyterlab/jupyterlab/pull/8850/commits/c4effe0900876a5afbc8340964f2ef0b29a7fc55

I just added exactly the packages I needed to get the example to compile. Are we sure that we need each one of these loaders, and terser?

Reply:
We need all the ones in the [base config](https://github.com/jupyterlab/jupyterlab/blob/master/buildutils/src/webpack.config.base.ts), and we'll need terser once we support production builds for extensions.

Reply:
We might not need `worker-loader`, but we added that due to build errors folks were seeing.

Reply:
We also might not explicitly need `to-string-loader`, since right now that is only used for codemirror.

Reply:
I needed to-string-loader when one of the example extensions imported codemirror, even if it didn't actually bundle codemirror.